starknet_py/abi/v1/model.py

```diff
@@ -35,10 +35,4 @@ class Event:
         str, StructType
     ]  #: Abi of structures and enums defined by the class.
     functions: Dict[str, Function]  #: Functions defined by the class.
-    constructor: Optional[
-        Function
-    ]  #: Contract's constructor. It is None if class doesn't define one.
-    l1_handler: Optional[
-        Function
-    ]  #: Handler of L1 messages. It is None if class doesn't define one.
```

Comment:
Why was this removed?

Reply:
There is no more `constructor` or `l1_handler` type.
They're all functions